src/igor/schedule.go

```diff
@@ -44,6 +44,9 @@ func areNodesFree(clusternodes []uint64, requestedindexes []int) bool {
 
 // Returns true if nodes[index] through nodes[index+count-1] are free (set to 0)
 func isFree(nodes []uint64, index, count int) bool {
```

Comment:
While this solves the panic, does it make sense to put the check this deep in the code? Shouldn't `runSub` do the check?